content/node/tutorials/zero-to-node/pocket-config.md

```diff
@@ -126,7 +126,7 @@ To create a new config file:
 1. Run the following command, which will create the default `config.json` file, add the seeds, set port 8081 to 8082, and increase the RPC timeout value:
```

Comment:
```suggestion
1. Run the following command, which will create the default `config.json` file for this version, add the current seeds (as of June 2023), and change the RPC default port from 8081 to 8082:
```
The latest config printed by `pocket util print-configs` prints a default RPC timeout of `30000`, this guide when written the default in the config was `10000` so it needed to be increased to `15000`, but the default now is `30000` so it would be cutting it in half decreasing it now, instead of increasing it 50%.